pyi.py

```diff
@@ -301,6 +301,7 @@ def visit_FunctionDef(self, node: ast.FunctionDef) -> None:
                     continue
             # allow "raise", a number of stubs have this
```

Comment:
Should instead say something like "Special-case raise because a previous version of flake8-pyi allowed it"